docker/domserver/Dockerfile

```
@@ -82,6 +82,10 @@ RUN useradd -m domjudge \
 	&& chmod 755 /scripts/start.sh \
 	&& chmod 755 /scripts/bin/* \
 	&& ln -s /scripts/bin/* /usr/bin/
+
+# Make the scripts available to the root user
+RUN echo "PATH=$PATH:/opt/domjudge/domserver/bin" >> /root/.bashrc
```

Using `ENV` to update the PATH might be cleaner(provided it works as described https://stackoverflow.com/a/38742545/841300)
```suggestion
ENV PATH="$PATH:/opt/domjudge/domserver/bin"
```

Another possible hack is to link the binaries to somewhere that is presumably already on the path:
```
RUN for prog in /opt/domjudge/domserver/bin/*; do ln -sf $prog /usr/local/bin; done
```

Both of these options will likely allow for an admin to do something like `docker run domserver <some_domjudge_binary> args` directly, whereas I'm not certain the bashrc hack will do.(You'd probably need to do something like `docker run domserver bash -c "<some_domjudge_binary> args"`)

---

I think the ENV option would be indeed better, did not know of this.

The other option feels almost like the build should use the `--enable-fhs` or the $domserver_bindir for the configure should be altered. Which I think is less tested.